submissions/kazeemErinfolami/aboutMe/html/index.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
```

Comment:
This file shouldn't be inside your `aboutMe` folder. Create an `html` folder inside your named folder and place the file inside that folder

Reply:
Same goes to others as well.